<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tt3::gui::PublicActivityManager</class>
 <widget class="QWidget" name="tt3::gui::PublicActivityManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="filterWidget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="filterLabel">
        <property name="text">
         <string>Filter:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="filterLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTreeWidget" name="publicActivitiesTreeWidget">
     <property name="contextMenuPolicy">
      <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="buttonsWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="4" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="modifyPublicActivityPushButton">
        <property name="text">
         <string>Modify public activity</string>
        </property>
        <property name="icon">
         <iconset resource="tt3-gui.qrc">
          <normaloff>:/tt3-gui/Resources/Images/Actions/ModifyPublicActivitySmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/ModifyPublicActivitySmall.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="createPublicActivityPushButton">
        <property name="text">
         <string>Create public activity</string>
        </property>
        <property name="icon">
         <iconset resource="tt3-gui.qrc">
          <normaloff>:/tt3-gui/Resources/Images/Actions/CreatePublicActivitySmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/CreatePublicActivitySmall.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="destroyPublicActivityPushButton">
        <property name="text">
         <string>Destroy public activity</string>
        </property>
        <property name="icon">
         <iconset resource="tt3-gui.qrc">
          <normaloff>:/tt3-gui/Resources/Images/Actions/DestroyPublicActivitySmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/DestroyPublicActivitySmall.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>filterLineEdit</tabstop>
  <tabstop>publicActivitiesTreeWidget</tabstop>
  <tabstop>createPublicActivityPushButton</tabstop>
  <tabstop>modifyPublicActivityPushButton</tabstop>
  <tabstop>destroyPublicActivityPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="tt3-gui.qrc"/>
 </resources>
 <connections/>
</ui>
